Gray Code.cpp

```diff
@@ -0,0 +1,55 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+void Gray_Code(int n)
+{
+    if(n <= 0)
+    {
+        cout << "Invalid input length";
+        return;
+    }
+    vector<string> v;
+    v.push_back("0");
+    v.push_back("1");
+    for(int i = 2; i < (1 << n); i <<= 1)
+    {
+        for(int j = i-1; j >= 0; j--)
+	    v.push_back(v[j]);
```

Comment:
inconsistency in indentation

Reply:
Can you please open the code file in my branch.I have properly indented it but it's not showing in the files changed section.Kindly see the actual file and then review.

Reply:
![Capture](https://user-images.githubusercontent.com/46227193/77884171-2fb61b00-7282-11ea-9b0d-7363bb1db70a.PNG)

Reply:
> 
> 
> ![Capture](https://user-images.githubusercontent.com/46227193/77884171-2fb61b00-7282-11ea-9b0d-7363bb1db70a.PNG)

This is the screenshot of the part of the code you told to change.

Reply:
@somya-kapoor @jainaman224 Please review

Reply:
![Screenshot (108)](https://user-images.githubusercontent.com/42409030/78183250-3496ec80-7485-11ea-811b-e7c7dd3fa615.png)

Reply:
@somya-kapoor Is it ok now?